core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -19,71 +19,64 @@
 
 var waitFor = require('./waitFor.js');
 var action = require('./action.js');
+const { element } = require('protractor');
 
 var LearnerDashboardPage = function() {
   var LEARNER_DASHBOARD_URL = '/learner-dashboard';
-  var completedSection =
-    element(by.css('.protractor-test-completed-section'));
-  var playLaterSection =
-      element(by.css('.protractor-test-play-later-section'));
-  var incompleteSection =
-    element(by.css('.protractor-test-incomplete-section'));
+  var homeSection =
+    element(by.css('.protractor-test-home-section'));
+  var goalsSection =
+    element(by.css('.protractor-test-goals-section'));
+  var progressSection =
+    element(by.css('.protractor-test-progress-section'));
+  var communityLessonsSection =
+    element(by.css('.protractor-test-community-lessons-section'));
   var feedbackSection =
     element(by.css('.protractor-test-feedback-section'));
   var feedbackThread =
     element(by.css('.protractor-test-feedback-thread'));
-  var completedCollectionsSection =
-    element(by.css('.protractor-test-completed-collection-section'));
-  var completedExplorationsSection =
-    element(by.css('.protractor-test-completed-exp-section'));
-  var incompleteCollectionsSection =
-    element(by.css('.protractor-test-incomplete-collection-section'));
-  var incompleteExplorationsSection =
-    element(by.css('.protractor-test-incomplete-exp-section'));
-  var subscriptionsSection =
-    element(by.css('.protractor-test-subscriptions-section'));
   var subscriptionName =
     element.all(by.css('.protractor-test-subscription-name'));
   var feedbackExplorationTitle =
     element.all(by.css('.protractor-test-feedback-exploration'));
   var feedbackMessage =
     element.all(by.css('.protractor-test-feedback-message'));
+  var addToLearnerGoalsButton =
+    element(by.css('.protractor-test-add-topic-to-current-goals-button'));
+  var currentGoalsTopic =
+    element(by.css('.protractor-test-topic-name-in-current-goals'));
+  var skillProficiencyTopic =
+    element(by.css('.protractor-test-skill-proficiency-topic-title'));
+  var completedGoalsTopic =
+    element(by.css('.protractor-test-completed-goals-topic-name'));
+  var topicNameInLearnerStorySummaryTiles =
+    element.all(by.css(
+      '.protractor-test-topic-name-in-learner-story-summary-tile'));
+  var storyNameInLearnerStorySummaryTiles =
+      element.all(by.css(
+        '.protractor-test-story-name-in-learner-story-summary-tile'));
+  var topicNameInLearnerTopicSummaryTiles =
+      element.all(by.css('.protractor-test-learner-topic-summary-tile-title'));
```

Comment:
Thanks for your changes! These aren't quite plural though:

```suggestion
  var topicNamesInLearnerStorySummaryTiles =
    element.all(by.css(
      '.protractor-test-topic-name-in-learner-story-summary-tile'));
  var storyNamesInLearnerStorySummaryTiles =
      element.all(by.css(
        '.protractor-test-story-name-in-learner-story-summary-tile'));
  var topicNamesInLearnerTopicSummaryTiles =
      element.all(by.css('.protractor-test-learner-topic-summary-tile-title'));
```

Reply:
Done.